src/screens/SendSheet.js

```diff
@@ -381,13 +386,18 @@ export default function SendSheet(props) {
   const onChangeAssetAmount = useCallback(
     newAssetAmount => {
       if (isString(newAssetAmount)) {
+        setLastFocusedInputHandle(assetInputRef);
         sendUpdateAssetAmount(newAssetAmount);
         analytics.track('Changed token input in Send flow');
       }
     },
-    [sendUpdateAssetAmount]
+    [sendUpdateAssetAmount, setLastFocusedInputHandle, assetInputRef]
   );
 
+  const handleCustomGasBlur = useCallback(() => {
+    lastFocusedInputHandle?.current?.focus();
+  }, [lastFocusedInputHandle]);
+
```

Comment:
there's no need for this wrapping. Why won't you directly use `lastFocusedInputHandle?.current?.focus`?